gotocompany/shield/v1beta1/admin.proto

```diff
@@ -0,0 +1,1159 @@
+syntax = "proto3";
+
+package gotocompany.shield.v1beta1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "validate/validate.proto";
+
+option go_package = "github.com/goto/proton/shield/v1;shieldv1beta1";
+option java_outer_classname = "Admin";
```

Comment:
Could keep this as Shield and same with `shield.proto`

Otherwise this will be breaking changes.